package.json

```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "mocha ./test/* --timeout 10000 --exit",
-    "start": "nodemon server.js"
+    "start": "node server.js",
+    "dev": "nodemon server.js"
```

Comment:
```suggestion
```
Local development won't have any effect on the Heroku deployment

Reply:
I think this is just for developers to be able to use Nodemon on their local repos right?